mypy/stubgen.py

```diff
@@ -926,7 +926,7 @@ def process_namedtuple(self, lvalue: NameExpr, rvalue: CallExpr) -> None:
         if self._state != EMPTY:
             self.add('\n')
         if isinstance(rvalue.args[1], StrExpr):
-            items = rvalue.args[1].value.split(" ")
+            items = rvalue.args[1].value.replace(',', ' ').split()
```

Comment:
Could you add a test if there are two commas in a row (e.g. someone makes a typo?

Reply:
Done - let me know if that's along the lines you were thinking or if I misunderstood you?

Reply:
Great, thank you!